pages/leaderboard/index.tsx

```diff
@@ -20,14 +21,17 @@ const LeaderboardPage = () => {
     <Layout title='LiveBetter | DBS Bank' heading='LiveBetter' user={userQuery.data}>
       <div className='mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8'>
         <h1 className='sr-only'>Leaderboard</h1>
-        <h2 className='text-md font-medium leading-6 text-gray-900'>Leaderboard</h2>
+        <h2 className='text-md font-bold leading-6 text-gray-900 mb-6'>Leaderboard</h2>
+
+        <div className='flex justify-evenly'>
+          <LeaderboardTopRankings />
+        </div>
+
         <div>
-          <br></br>
-          <br></br>
           <br></br>
```

Comment:
I think can remove this too haha

Reply:
@remuskwan Pushed a new update for the requested changes!